theme_vehicle/views/customizations.xml

```diff
@@ -1,171 +1,352 @@
-        <attribute name="data-oe-shape-data">{"shape":"web_editor/Airy/06","flip":[]}</attribute>
-    </xpath>
-    <xpath expr="//div[hasclass('container')]" position="before">
-        <div class="o_we_shape o_web_editor_Airy_06"/>
-    </xpath>
+    <!-- ======== TEXT - IMAGE ======== -->
+    <template id="s_text_image" inherit_id="website.s_text_image" name="Vehicle s_text_image">
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="pb24 pt56 o_cc o_cc1" remove="pt32 pb32" separator=" "/>
+        </xpath>
+        <xpath expr="//img" position="attributes">
+            <attribute name="src">/web_editor/image_shape/website.s_text_image_default_image/web_editor/float/flt_square_3.svg?c1=o-color-1&amp;c5=o-color-5</attribute>
+            <attribute name="data-shape">web_editor/float/flt_square_3</attribute>
+            <attribute name="data-original-mimetype">image/jpeg</attribute>
+            <attribute name="data-file-name">s_text_image.svg</attribute>
+            <attribute name="data-shape-colors">o-color-1;;;;o-color-5</attribute>
+        </xpath>
+        <xpath expr="//div[hasclass('row')]/div[2]" position="attributes">
+            <attribute name="class" add="col-lg-5 offset-lg-1" remove="col-lg-6" separator=" "/>
+        </xpath>
+        <xpath expr="//h2" position="before">
+            <h4 class="text-o-color-1"><b>The world is yours</b></h4>
+        </xpath>
+        <xpath expr="//h2" position="replace" mode="inner">
+            No compromise
+        </xpath>
+        <xpath expr="//p" position="replace" mode="inner">
+            Performance and design were the key words during the conception of this super car. No compromises were made to keep the pleasure of driving despite those constraints.
+        </xpath>
+        <xpath expr="//p[2]" position="replace" mode="inner">
+            Take advantage of a free track trial to discover it.
+        </xpath>
+    </template>
 
-    <xpath expr="//div[hasclass('row')]/div" position="attributes">
-        <attribute name="class" add="col-lg-5" remove="col-lg-6" separator=" "/>
-    </xpath>
-    <xpath expr="//div[hasclass('row')]/div[2]" position="attributes">
-        <attribute name="class" add="offset-lg-1" separator=" "/>
-    </xpath>
+    <!-- ======== IMAGE - TEXT ======== -->
+    <template id="s_image_text" inherit_id="website.s_image_text" name="Vehicle s_image_text">
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="pb56 pt24 o_cc o_cc1" remove="pt32 pb32" separator=" "/>
+        </xpath>
+        <xpath expr="//img" position="attributes">
+            <attribute name="src">/web_editor/image_shape/website.s_image_text_default_image/web_editor/float/flt_square_2.svg?c2=o-color-1&amp;c5=o-color-5</attribute>
+            <attribute name="data-shape">web_editor/float/flt_square_3</attribute>
```

Comment:
flt_square_2 used in the URL, flt_square_3 in the data-shape... :| Makes edition broken.